<!DOCTYPE html>
<html>
<head>
    <meta http-equiv="X-UA-Compatible" content="IE=9">
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>Cancer Regulome :: Software</title>

    <link href="http://fonts.googleapis.com/css?family=Pontano+Sans|Gentium+Basic" rel="stylesheet" type="text/css">
    <link href="styles/csacr.css" rel="stylesheet">
    <link href="styles/csacr-responsive.css" rel="stylesheet">
    <link rel="shortcut icon" href="favicon.ico">

    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
    <script src="scripts/html5.js"></script>
    <link href="styles/csacr.ie.css" rel="stylesheet">
    <![endif]-->
    <script src="scripts/jquery.js"></script>
    <script src="scripts/bootstrap.js"></script>

    <script type="text/javascript" src="scripts/google_analytics.js"></script>
    <script type="text/javascript" src="scripts/menus_loader.js"></script>
</head>
<body>
<div class="container">
<div id="menusContainer"></div>
<div id="mainContainer">
    <a name="applications"><br/><br/></a>
    <div class="row">
        <div class="span9">
            <div class="page-header">
                <h2>Applications</h2>
            </div>
            <div>
                <h4>Regulome Explorer</h4>
                <div>
                    This web-based tool provides for the exploration of data from TCGA. It can be used to explore the following:
                    <ul>
                        <li>The cancer comparison data sets which give information about common gene disruptions across different cancers.</li>
                        <li>Associations among multiple types of features (e.g., gene expression, methylation, copy number variation, clinical), computed using
                            random forest regression and displayed within an interactive Circvis tool.</li>
                        <li>Individual genome aberrations identified through the analysis of structural variations using FastBreak.</li>
                    
                    Read more about Regulome Explorer in its <a href="http://regulome-explorer.googlecode.com" target="_blank">open source project</a>.
                    <br/>
                </div>
            </div>
            <hr/>            
            <div>
                <h4>Colorectal Cancer Aggressiveness Explorer</h4>
                <div>
                    The CRC Aggressiveness Explorer allows the exploration of molecular signatures associated with aggressive CRC. A red data point implies that
                    the signature is more prevalent in tumors with aggressive colorectal cancer, while a blue data point indicates the opposite. A color gradient
                    is used for the strength of association.
                </div>
            </div>
        </div>
        <div class="span3">
            <div class="well sideinfo">
                <div class="sideinfo-heading">
                    <h4>Web Applications</h4>
                </div>
                <ul>
                     <li><a href="http://explorer.cancerregulome.org/all_pairs/" target="_blank"> Regulome Explorer: All Pairs Analysis</a></li>
                     <li><a href="http://explorer.cancerregulome.org/crc_agg/" target="_blank">CRC Aggressiveness Explorer</a></li>
                </ul>
            </div>
        </div>
    </div>
    <a name="data_analysis"><br/><br/></a>
    <div class="row">
        <div class="span12">
            <div class="row">
                <div class="span9">
                    <div class="page-header">
                        <h2>Data Analysis</h2>
                    </div>
                    <div>
                        <h4>Fastbreak</h4>
                        <div>
                            This application has been designed to quickly screen for structural variation. We have now expanded Fastbreak to begin looking for
                            patterns of structural variation across cancer types. A paper covering the Fastbreak algorithm is submitted for publication.
                            <br/>
                            <br/>
                            Read more about Fastbreak in its <a href="http://fastbreak.googlecode.com" target="_blank">open source project</a>.
                        </div>
                    </div>
                    <hr/>
                    <div>
                        <h4>RF-ACE</h4>
                        <div>
                            This program is an efficient C++ implementation of a robust machine learning algorithm for uncovering multivariate associations from
                            large and diverse data sets. RF-ACE natively handles numerical and categorical data with missing values, and potentially large
                            quantities of noninformative features.
                            <br/>
                            <br/>
                            Read more about RF-ACE in its <a href="http://rf-ace.googlecode.com" target="_blank">open source project</a>.
                        </div>
                    </div>
                    <hr/>
                </div>
            </div>
        </div>
    </div>
</div>
<footer>
    <div>&copy; 2016, Institute for Systems Biology, All Rights Reserved</div>
</footer>
</div>
</body>
</html>
